src/stores/OwnProfileStore.ts

```diff
@@ -100,6 +100,7 @@ export class OwnProfileStore extends AsyncStoreWithClient<IState> {
     }
 
     protected async onNotReady(): Promise<void> {
+        this.onProfileUpdate.cancel();
```

Comment:
This turns out to be needed because otherwise, a test would complete, and `onProfileUpdate` would fire halfway through the *next* test, potentially causing errors.